app/com/m3/octoparts/aggregator/service/PartResponseLocalContentSupport.scala

```diff
@@ -0,0 +1,29 @@
+package com.m3.octoparts.aggregator.service
+
+import com.m3.octoparts.aggregator.PartRequestInfo
+import com.m3.octoparts.model.PartResponse
+import com.m3.octoparts.model.config.{ HttpPartConfig, ShortPartParam }
+
+import scala.concurrent.Future
+
+trait PartResponseLocalContentSupport extends PartRequestServiceBase {
+
+  override def processWithConfig(ci: HttpPartConfig,
+                                 partRequestInfo: PartRequestInfo,
+                                 params: Map[ShortPartParam, Seq[String]]): Future[PartResponse] = {
+    if (ci.localContentsEnabled) {
```

Comment:
OK. This is different from the image I had (a la Hystrix fallback). Anyways, this should be documented.